res/mime.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<mime-info xmlns="http://www.freedesktop.org/standards/shared-mime-info">
+  <mime-type type="audio/x-fur">
+    <comment>Furnace module</comment>
+    <magic priority="50">
+      <match type="string" value="-Furnace module-" offset="0" />
+      <match type="string" value="\170\234" offset="0" />
```

Comment:
Warning - this will result in all zlib-compressed files being treated as Furnace modules.

Reply:
I was concerned about that as well, but I did a quick run-through of every gzipped file I have in my downloads folder (including related formats like .vgz) and it turns out that .fur modules are the outlier. Nearly every other gzipped file I've come across starts with 0x1F8B, while gzipped .fur modules seem to start with 0x789C.

Reply:
Oh wait never mind, I'm dumb; it's zlib, not gzip. Will attempt to fix.

Reply:
I've decided to comment out that line for now; it seems like most file managers will still recognize it by the extension alone, and as far as I can tell there isn't really a better way to reliably detect header data in zlib compressed files without calling an external utility.